src/main/java/core/basesyntax/service/impl/ReturnAction.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ActionHandler;
+
+public class ReturnAction implements ActionHandler {
+    public void apply(String fruit, int quantity) {
```

Comment:
According to the checklist, the `apply` method should be overriding a method from the `ActionHandler` interface. Please ensure that you use the `@Override` annotation to make it clear that this method is overriding a method from its interface.